Content/Items/BossDomain/MechDrive.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Terraria.ID;
+
+namespace PathOfTerraria.Content.Items.BossDomain;
+
+internal class MechDrive : ModItem
+{
+	public override void SetDefaults()
+	{
+		Item.CloneDefaults(ItemID.Silk);
+		Item.Size = new Vector2(26, 20);
+	}
```

Comment:
Not seeing this used for any functionality, unless I missed something. What is this intended to do?